.travis.yml

```diff
@@ -237,6 +322,28 @@ jobs:
         - cmake --build build -- -j4
       script: (cd build; ctest -V -L CORE -j2)
 
+    # Ubuntu Linux with glibc using g++-5, disable USE_DSTRING
+    - stage: Test different OS/CXX/Flags
+      os: linux
+      sudo: false
+      compiler: gcc
+      cache: ccache
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - libwww-perl
+            - g++-5
+            - libubsan0
+            - parallel
+      before_install:
+        - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
+      # env: COMPILER=g++-5 SAN_FLAGS="-fsanitize=undefined -fno-sanitize-recover -fno-omit-frame-pointer"
+      env:
+        - NAME="USE_STD_STRING"
+        - COMPILER="ccache /usr/bin/g++-5"
+        - EXTRA_CXXFLAGS="-D_GLIBCXX_DEBUG -DUSE_STD_STRING"
```

Comment:
Previously, the USE_STD_STRING builds were not running tests, with this patch hunk they will now be running tests (which appear to fail) - is this an intentional change? If so, we need the tests and/or USE_STD_STRING fixed before merging. :-)